### PR TITLE
fasttree 2.1.10

### PR DESCRIPTION
**Pull request by tseemann**
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [x] Written a sensible test? (The best test is to compile and run a sample program.)

### Updates to existing formula: have you

- [x] Not altered the `bottle` section?
- [x] Checked that the tests still pass?

### CODE REVIEW

**Comment by tseemann** [comment]
Once this gets accepted @sjackman I will fix `gubbins` and `nullarbor` (on mine) to remove the `--with-double` dependency.

File: fasttree.rb [hunk 1]
```diff
@@ -39,6 +45,6 @@ def install
       >3
       LCLYTHIGRNIYYGSYLYSETWNTGIMLLLITMATAFMGTTLPWGQMSFWGATVITNLFSAIPYIGTNLV
     EOF
-    assert_match(/1:0.\d+,2:0.\d+,3:0.\d+/, `#{bin}/FastTree test.fa`)
+    assert_match /1:0.\d+,2:0.\d+,3:0.\d+/, shell_output("#{bin}/FastTree #{testpath}/test.fa 2>&1")
```
Comment (sjackman) [inline fasttree.rb]: `testpath` is implied. No need to mention it explicitly.

File: fasttree.rb [hunk 2]
```diff
@@ -15,7 +16,11 @@ class Fasttree < Formula
     sha256 "642dfb0168fb2ebfcb8314206da71aa9d4aa70c7d4ee7d77fe1a882e2f4e0c3b" => :x86_64_linux
   end
 
-  option "with-double", "Use double precision maths for accurate branch lengths (disables SSE)"
+  # 26 Aug 2017; Community mostly wants USE_DOUBLE; make it default now
+  # http://www.microbesonline.org/fasttree/#BranchLen
+  # http://darlinglab.org/blog/2015/03/23/not-so-fast-fasttree.html
+
+  option "without-double", "Disable double precision maths (disables SSE)"
```
Comment (sjackman) [inline fasttree.rb]: Please change the comment to
`Disable double precision floating point. Use single precision floating point and enable SSE.`

**Comment by jonchang** [comment]
Merged. Thanks @tseemann!

**Pull request closed without merging**